sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -103,121 +114,110 @@ - (NSURLRequest *)buildRequestForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     }
 }
 
-- (NSDictionary *)requestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
+- (NSDictionary *)openRTBRequestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     NSMutableDictionary *requestDict = [[NSMutableDictionary alloc] init];
 
-    if (self.primaryAdServer == PBPrimaryAdServerMoPub || self.primaryAdServer == PBPrimaryAdServerUnknown) {
-        requestDict[@"cache_markup"] = @(1);
-    }
+    requestDict[@"id"] = @"some-request-id";
+    requestDict[@"test"] = @(1);
+    requestDict[@"tmax"] = @(500);
 
-    requestDict[@"sort_bids"] = @(1);
-    // need this so DFP targeting keys aren't too long
-    requestDict[@"max_key_length"] = @(20);
-    requestDict[@"account_id"] = self.accountId;
-    requestDict[@"tid"] = [[NSUUID UUID] UUIDString];
-    requestDict[@"prebid_version"] = @"0.21.0-pre";
+    requestDict[@"app"] = [self openrtbApp];
+    requestDict[@"device"] = [self openrtbDevice];
+    requestDict[@"user"] = [self openrtbUser];
+    requestDict[@"geo"] = [self openrtbGeo];
```

Comment:
Geo object should be nested under device object, see Android implementation.

Reply:
The geo object was added in 2 places... removed this specific code